app/store/env/go.js

```diff
@@ -2236,13 +2236,15 @@ YUI.add('juju-env-go', function(Y) {
 
       @method getChangeSet
       @param {String} bundleYAML The bundle YAML file contents.
+      @param {String} changesToken The token identifying a bundle change set
```

Comment:
this reads that the bundleYAML is ignored if you have a changestoken, but in the if block below we first run with the yaml if it's not null?